assets/samplesheet.csv

```diff
@@ -0,0 +1,3 @@
+sample,fastq_1,fastq_2
```

Comment:
This is not updated, please remove or update

Reply:
Hmm, what do you mean? I just checked with the [pipeline template](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/assets/samplesheet.csv), to me it seems the samplesheet is up-to-date

Reply:
yes of course the template contains an example samplesheet, but this should be adapted to reflect the samplesheet that should be used to run the pipeline. It is an example samplesheet for running the pipeline. As you have already a samplesheet example here: https://github.com/qbic-pipelines/rnadeseq/blob/master/testdata/Sample_preparations.tsv
You can either remove the assets/samplesheet.csv or copy the Sample_preparation.tsv here.